cmd/scw/testdata/test-all-usage-k8s-usage.golden

```diff
@@ -6,13 +6,13 @@ USAGE:
   scw k8s <command>
 
 AVAILABLE COMMANDS:
-  acl          Access Control List (ACL) management commands
-  cluster      Kapsule cluster management commands
-  cluster-type Cluster type management commands
-  kubeconfig   Manage your Kubernetes Kapsule cluster's kubeconfig files
-  node         Kapsule node management commands
-  pool         Kapsule pool management commands
-  version      Available Kubernetes versions commands
+  acl             Access Control List (ACL) management commands
+  cluster         Kapsule cluster management commands
+  cluster-type    Cluster type management commands
+  kubeconfig      Manage your Kubernetes Kapsule cluster's kubeconfig files
+  node            Kapsule node management commands
+  pool            Kapsule pool management commands
+  version         Available Kubernetes versions commands
```

Comment:
I need to rebuild this golden to allow checks.
The doc generator have a bug on indentation with hidden commands